connect.json

```diff
@@ -0,0 +1,74 @@
+{
+  "cases": [
+    {
+      "description": "an empty board has no winner",
+      "board": [
+        ". . . . .",
+        ". . . . .",
+        ". . . . .",
+        ". . . . .",
+        ". . . . ."
+      ],
+      "expected": ""
+    },
+    {
+      "description": "only edges does not make a winner",
+      "board": [
+        ". . . . .",
+        ". . . . X",
+        ". O . X .",
+        ". . O . .",
+        ". . . O ."
+      ],
+      "expected": ""
+    },
+    {
+      "description": "Rectangle black - black wins",
+      "board": [
+        ". O . .",
+        "O X X X",
+        "O X O .",
+        "X X O X",
+        ". O X ."
+      ],
+      "expected": "X"
+    },
+    {
+      "description": "Rectangle white - white wins",
+      "board": [
+        ". O . .",
+        "O X X X",
```

Comment:
It becomes hard to visualise the board when it i presented like this instead of

```
               . O . .
                O X X X
                 O X O .
                  X X O X
                   . O X .
```

The extra spaces make it clear that O has won, which isn't obvious (to me at least) without the extra spaces